django/contrib/auth/management/__init__.py

```diff
@@ -66,7 +66,7 @@ def create_permissions(app_config, verbosity=2, interactive=True, using=DEFAULT_
     except LookupError:
         return
 
-    if not router.allow_migrate(using, Permission):
+    if not router.allow_migrate(using, 'auth', Permission):
```

Comment:
I'll try to post something on the ML tomorrow but I may miss the mark as I'm travelling, in the meantime I've added my thoughts to the ticket so someone can take it from there if needed.